gravatar/src/main/java/com/gravatar/services/IdentityService.kt

```diff
@@ -0,0 +1,93 @@
+package com.gravatar.services
+
+import com.gravatar.logger.Logger
+import com.gravatar.restapi.models.Identity
+import com.gravatar.restapi.models.SelectAvatar
+import kotlinx.coroutines.withContext
+import okhttp3.OkHttpClient
+import com.gravatar.di.container.GravatarSdkContainer.Companion.instance as GravatarSdkDI
+
+/**
+ * Service for managing Gravatar identities.
+ */
+public class IdentityService(private val okHttpClient: OkHttpClient? = null, oauthToken: String) {
```

Comment:
I have doubts about using the name Identity. We don't use that term widely so maybe we should look for another name.

Reply:
As long as we keep the naming consistent with the backend I'm ok with this one.